unpackbootimg.c

```diff
@@ -191,7 +191,7 @@ int main(int argc, char **argv)
     fseek(f, i, SEEK_SET);
     if(fread(&header, sizeof(header), 1, f)) {};
 
-    if(magic == BOOT_MAGIC) {
+    if(strcmp(magic, BOOT_MAGIC) == 0) {
```

Comment:
This is how strcmp is previously used in the code, which is also correct. Stylistically you should match it for consistency:

https://github.com/osm0sis/mkbootimg/blob/master/unpackbootimg.c#L136

Reply:
i added that already to my commit.
Either use
strcmp(str1, str2) == 0
or !strcmp(str1, str2) 
both are correct

Reply:
I know, I said both are correct, but I also told you multiple times how it should be to match the existing coding style.

Reply:
done sir

Reply:
Still looks like == 0 to me. Change it to !strcmp to match correctly. Thanks.